content/tutorials/manage-the-rippled-server/troubleshooting/understanding-log-messages.md

```diff
@@ -4,6 +4,28 @@ The following sections describe some of the most common types of log messages th
 
 This is an important step in [Diagnosing Problems](diagnosing-problems.html) with `rippled`.
 
+## Log Message Structure
+
+The following shows the format of the log file:
+
+```text
+2020-Jul-08 20:10:17.372178946 UTC Peer:WRN [236] onReadMessage from n9J2CP7hZypxDJ27ZSxoy4VjbaSgsCNaRRJtJkNJM5KMdGaLdRy7 at 197.27.127.136:53046: stream truncated
+2020-Jul-08 20:11:13.582438263 UTC PeerFinder:ERR Logic testing     52.196.126.86:13308 with error, Connection timed out
+2020-Jul-08 20:11:57.728448343 UTC Peer:WRN [242] onReadMessage from n9J2CP7hZypxDJ27ZSxoy4VjbaSgsCNaRRJtJkNJM5KMdGaLdRy7 at 197.27.127.136:53366: stream truncated
+2020-Jul-08 20:12:12.075081020 UTC LoadMonitor:WRN Job: sweep run: 1172ms wait: 0ms
+```
+
+Each line represents one log entry, with the following parts in order, separated by spaces:
+
+1. The date the log entry was written, such as `2020-Jul-08`.
+2. The time the log entry was written, such as `20:12:12.075081020`.
+3. The time zone indicator `UTC`. (Log dates are always in UTC.) [New in: rippled 1.5.0][]
+4. The log partition and severity, such as `LoadMonitor:WRN`.
+5. The log message, such as `Job: sweep run: 1172ms wait: 0ms`.
```

Comment:
Aside: I list the items as divided by spaces (instead of listing the date/time/zone as one item) because I figure it's likely that system admins will use a tool like `awk` to read the logs and split the components by spaces.

Reply:
This section looks good to me!